modules/tensor_mechanics/examples/bridge/bridge.i

```diff
@@ -0,0 +1,146 @@
+[GlobalParams]
+  disp_x = disp_x
+  disp_y = disp_y
+  disp_z = disp_z
+[]
+
+[Mesh]
+  displacements = 'disp_x disp_y disp_z'
+  type = FileMesh
+  file = bridge.e
+[]
+
+[Variables]
+  [./disp_x]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./disp_y]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./disp_z]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[Kernels]
+  [./gravity_y]
+    type = GravityTM
+    variable = disp_y
+    value = -9.81
+  [../]
+  [./TensorMechanics]
+  [../]
+[]
+
+[AuxVariables]
+  [./von_mises]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
+
+[AuxKernels]
+  [./von_mises_kernel]
+    type = RankTwoScalarAux
+    variable = von_mises
+    rank_two_tensor = stress
+    execute_on = timestep_end
+    scalar_type = VonMisesStress
+  [../]
+[]
+
+[BCs]
+  [./PressureTM]
+    [./load]
+      boundary = 1
+      factor = 5e5 # Pa
+    [../]
+  [../]
+  [./anchor_x]
+    type = PresetBC
+    variable = disp_x
+    boundary = '2 3 4 5 6'
```

Comment:
It may be worthwhile to give all these boundaries names. You could dust off the rarely used on-the-fly names support. Just a suggestion